api/invite.go

```diff
@@ -399,9 +402,21 @@ func (a *Api) SendInvite(res http.ResponseWriter, req *http.Request, vars map[st
 			//None exist so lets create the invite
 			invite, _ := models.NewConfirmationWithContext(models.TypeCareteamInvite, models.TemplateNameCareteamInvite, invitorID, ib.Permissions)
 
+			// if the invitee is already a Tidepool user, we can use his preferences
 			invite.Email = ib.Email
 			if invitedUsr != nil {
 				invite.UserId = invitedUsr.UserID
+
+				// let's get the invitee user preferences
+				inviteePreferences := &models.Preferences{}
```

Comment:
Looks like a block of code (411 - 419) common between invite.go and forgot.go. 
Isn't it something worth to factorize?

Reply:
I think we could have a discussion on this one but at the bottom I dont think it is worth investigating a possible refactor here